master/master.cfg

```diff
@@ -551,6 +560,7 @@ arch_slaves = \
     ubuntu16_i386_slave + \
     debian8_arm_slave + \
     debian10_arm64_slave + \
+    debian11_arm64_slave + \
```

Comment:
Previously we just kept one builder per architecture to help catch anything major for that architecture.  But since these builds are quick I think we can keep both in this case.  It'll be nice to verify both kernels.